internal/credential/static/public_ids.go

```diff
@@ -10,18 +10,20 @@ import (
 )
 
 func init() {
-	if err := subtypes.Register(credential.Domain, Subtype, CredentialStorePrefix); err != nil {
+	if err := subtypes.Register(credential.Domain, Subtype, CredentialStorePrefix, PreviousCredentialStorePrefix); err != nil {
 		panic(err)
 	}
-	if err := subtypes.Register(credential.Domain, UsernamePasswordSubtype, CredentialPrefix); err != nil {
+	if err := subtypes.Register(credential.Domain, UsernamePasswordSubtype, CredentialPrefix, PreviousCredentialPrefix); err != nil {
 		panic(err)
 	}
 }
 
 // PublicId prefixes for the resources in the static package.
 const (
-	CredentialStorePrefix = "cs"
-	CredentialPrefix      = "cred"
+	CredentialStorePrefix         = "csst"
+	PreviousCredentialStorePrefix = "cs"
+	CredentialPrefix              = "credst"
```

Comment:
This credential is actually referring to the `username_password` credential, each type should get it's own prefix as they have different attributes. So something like `cup` or `credup`